cards.py

```
@@ -0,0 +1,223 @@
+
+
+def derive_body(blocks: List[Dict[str, Any]]) -> str:
+    """Render blocks to the plaintext `body` older readers still expect.
+
+    Deliberately reproduces the "Label:\\n...\\n\\nLabel:\\n..." shape that
+    members were building by hand, so the derived body is not a
+    regression for anything reading it today."""
+    return "\n\n".join(_render_block(b) for b in blocks if b)
+
+
+def normalize_actions(actions: Any) -> List[Dict[str, Any]]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `actions`

(ANN401)